scripts/gha/build_testapps.py

```diff
@@ -400,15 +414,15 @@ def _generate_makefiles_from_repo(sdk_dir):
 # build required ios frameworks based on makefiles
 def _build_ios_framework_from_repo(sdk_dir, api_config):
```

Comment:
docstring... Looks like api_config needs to have a `frameworks` property.

Reply:
Added one-liner docstrings.